doc/branding.md

```diff
@@ -72,15 +72,30 @@ With the configuration set as above, the title will be: `TeskaLabs | Page Title`
 
 Every image HAS TO be provided in SVG (vectorized).
 Use of pixel formats (PNG, JPG, ...) is strongly discouraged.
+While creating the branding images, use full width/height of the canvas (ratio 3:1 on full and 1:1 on minimized version). No padding is required for optimal viewing experience.
 
 ### Branding images
 
-Top-left location.
+format: SVG
```

Comment:
@petrKavulok Not sure if it is written by markdown standards. What I mean is that for new lines you need to have an empty line between the text. For indentations its better to use some bullet list. Pls update accordingly and pls take a look how it looks like when rendered as markdown (I bet visualstudio has some package for it)